app/init/db_init.py

```diff
@@ -0,0 +1,48 @@
+from passlib.context import CryptContext
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+from models import User, Operator
+
+passw = "p!32Mz?Wt59X~[kC"
+url = f'postgresql://postgres:postgres@/postgres?host=postgres'
+engine = create_engine(url, pool_pre_ping=True)
+SessionLocal = sessionmaker(autocommit=False, autoflush=False, bind=engine)
+
+db = SessionLocal()
+
+
+def init_admin_user():
+    pwd_context = CryptContext(schemes=["argon2"])
+    password_hashed = pwd_context.hash("Windows8")
+    basic_admin = User(email="stefan.drobic@vizlore.com", password=password_hashed)
+
+    db.add(basic_admin)
+    db.commit()
+
+# def init_operator (>, <, ...)
+def init_operators():
```

Comment:
Why are you adding these operators in the db initialization?

Reply:
These operators are used when defining a rule in pdm, for example a rule where:
TEMP > 50c AND HUMID < 20%
would be described using the ids of the units (TEMP, HUMID), ids of the operators (i.e. 1, 3) and the values as numbers.
Because there is a set of them, I just add them during app deployment, or rather during the entrypoint.sh as of now.

Reply:
ok